mods/ra/maps/allies-06a/allies06a.lua

```diff
@@ -115,6 +115,19 @@ CaptureRadarDome = function()
 
 	Trigger.OnCapture(Radar, function()
 		player.MarkCompletedObjective(CaptureRadarDomeObj)
+		Beacon.New(player, TechLab1.CenterPosition)
```

Comment:
Please notice that the Tech Centers are only revealed if the mission difficulty isn't hard, so it would be nice if you could move the beacons as well as the message into the `if`-check.

Reply:
Please let me elaborate. I did that on purpose to have a feedback if the player captures the radar dome on hard difficulty level. So, the player acquires the coordinations of the tech centers. Thereby, the beacons are placed for 30 seconds, which are also seen on the minimap. However, the shroud doesn't get revealed. Keeping the mission the same as before. Otherwise there is no point in capturing the radar dome on hard difficulty level.

But please feel free to say that you don't like it, and I will gladly change it.

Reply:
Ok. Let's give it a try and see if the players like it ;).